test/conformance/tests/sriov_operator.go

```diff
@@ -913,9 +913,72 @@ var _ = Describe("[sriov] operator", func() {
 				createUnschedulableTestPod(testNode, []string{sriovNetworkName}, resourceName)
 			})
 		})
+
+		Context("SRIOV and macvlan", func() {
+			// 25834
+			It("Should be able to create a pod with both sriov and macvlan interfaces", func() {
+
+				resourceName := "sriovnic"
+				sriovNetworkName := "sriovnetwork"
+				testNode := sriovInfos.Nodes[0]
+
+				sriovDevice, err := sriovInfos.FindOneSriovDevice(testNode)
+
+				Expect(err).ToNot(HaveOccurred())
+				createSriovPolicy(sriovDevice.Name, testNode, 5, resourceName)
+
+				ipam := `{"type": "host-local","ranges": [[{"subnet": "1.1.1.0/24"}]],"dataDir": "/run/my-orchestrator/container-ipam-state"}`
+				err = network.CreateSriovNetwork(clients, sriovDevice, sriovNetworkName, namespaces.Test, operatorNamespace, resourceName, ipam)
+				Expect(err).ToNot(HaveOccurred())
+				Eventually(func() error {
+					netAttDef := &netattdefv1.NetworkAttachmentDefinition{}
+					return clients.Get(context.Background(), runtimeclient.ObjectKey{Name: "sriovnetwork", Namespace: namespaces.Test}, netAttDef)
+				}, 10*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+
+				macvlanNadName := "macvlan-nad"
+				nodeNicName := sriovInfos.States[testNode].Status.Interfaces[0].Name
+				macvlanNad := network.CreateMacvlanNetworkAttachmentDefinition(macvlanNadName, namespaces.Test, nodeNicName)
+				defer clients.Delete(context.Background(), &macvlanNad)
+				err = clients.Create(context.Background(), &macvlanNad)
+				Expect(err).ToNot(HaveOccurred())
+				Eventually(func() error {
+					netAttDef := &netattdefv1.NetworkAttachmentDefinition{}
+					return clients.Get(context.Background(), runtimeclient.ObjectKey{Name: macvlanNadName, Namespace: namespaces.Test}, netAttDef)
+				}, 10*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+
+				createdPod := createTestPod(testNode, []string{sriovNetworkName, macvlanNadName})
+
+				nics, err := network.GetNicsByPrefix(createdPod, "net")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(len(nics)).To(Equal(2), "Pod should have two multus nics.")
+
+				stdout, _, err := pod.ExecCommand(clients, createdPod, "ethtool", "-i", "net1")
+				Expect(err).ToNot(HaveOccurred())
+				sriovVfDriver := getDriver(stdout)
+				Expect(cluster.IsDriverSupported(sriovVfDriver[:len(sriovVfDriver)-2])).To(BeTrue())
+
+				stdout, _, err = pod.ExecCommand(clients, createdPod, "ethtool", "-i", "net2")
+				macvlanDriver := getDriver(stdout)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(macvlanDriver).To(Equal("macvlan"))
+
+			})
+		})
 	})
 })
 
+func getDriver(ethtoolstdout string) string {
```

Comment:
can you move this to the utils folder and make the function return `(string,error)`

Reply:
Done